### ISSUE TITLE
Clarification is needed in the chapter "How should Adam’s hyperparameters be tuned?"

### ISSUE THREAD

**Issue by 21kc-caracol** [google-research/tuning_playbook#71]
![Screenshot_20240626-063650](https://github.com/google-research/tuning_playbook/assets/45002094/592c340c-9bb4-442a-8614-3012aa114a74)

Please clarify if For a budget of 10-25:
1. First tune the learning rate, then beta1.
2. (Or) Create a search space for both parameters, then run experiments to find the best combination.
An example will be appreciated.

I understood that its option 1. First tune for best learning rate, then fix that value, and start tuning beta1.

**Comment by laurahsisson** [comment]
It’s the second option. If you only have a limited number of trials, you can focus on just tuning the learning rate, but if you have more compute/time, you can optimize beta1 and LR in conjunction. 

If you change beta1 (or beta2 etc), you’ll need to retune LR.
